docs/roadmap.md

```diff
@@ -12,8 +12,8 @@ Your first consideration should be the risk you are taking. Don't worry, nobody
 Linux and by following the guides closely you should be able to get everything working even without a lot of knowledge as well.
 Still in case something goes wrong we are not responsible for it, we will try our best to help out however.
 
-Note that while its technically possible to remove macOS entirely, its strongly encouraged to dual boot it with Linux to have
-a backup in case something goes wrong.
+!!! warning "Removing macOS"
+           Note that while its technically possible to remove macOS entirely, its strongly encouraged to dual boot it with Linux to have a backup in case something goes wrong. We shall also recommend [creating a bootable macOS installer](https://support.apple.com/en-us/HT201372) so as to have an option of restoring macOS back.
```

Comment:
Keeping macOS is also good for firmware updates.

Also for wiki style ([this](https://en.m.wikipedia.org/wiki/Wikipedia:Manual_of_Style#First-person_pronouns) is better at explaining what I mean), "We shall also recommend" could be passive voice (i.e. "It is also recommended to create a..."). 

You could mention that internet recovery can be slow and unreliable (which is the reason that creating a macOS installer is useful ).

Reply:
Removing all the "shall" wording is probably better scoped for a separate pr if that's something we want to pursue